packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -684,6 +728,24 @@ export abstract class BaseService extends Resource
     };
   }
 
+  /**
+   *   Add an alarm based on ECS metric
```

Comment:
Please add more description of what this method does. What are the default values? What is the behavior when someone just calls `createEcsMetricAlarm(EcsMetric.CPU_UTILIZATION)`? If the default value is "1" for the threshold, calling this method will cause literally every deployment to fail because CPU utilization percent is going to be greater than 1%. We should make sure to provide good defaults for CPU/Memory utilization especially.